docs/source/guides/user/chapters/configuring.rst

```diff
@@ -174,59 +137,119 @@ of precedence for variables (from least precedence to most):
 So the least important value comes from the library or test code default, going
 all the way up to the test parameters system.
 
-Avocado Data Directories
-------------------------
+Supported data types when configuring Avocado
+---------------------------------------------
+
+As already said before, Avocado allows users to use both: configuration files
+and command-line options to configure its behavior. It is important to have a
+very well defined system type for the configuration file and argument options.
+
+Although config files options and command-line arguments are always considered
+``strings``, you should give a proper format representation so those values can
+be parsed into a proper type internally on Avocado.
+
+Currently Avocado supports the following data types for the configuration options:
+``string``, ``integer``, ``float``, ``bool`` and ``list``. Besides those
+primitives data types Avocado also supports custom data types that can be used
+by a particular plugin.
+
+Bellow, you will find information on how to set options based on those basic
+data types using both: configuration files and command-line arguments.
+
+Strings
+~~~~~~~
+
+Strings are the basic ones and the syntax is the same for both: configuration
+files and command-line arguments: Just the string that can be inside ``""`` or
+``''``.
+
+Example using the configuration file:
+
+.. code-block:: ini
+
+  [foo]
+  bar = 'hello world'
 
-When running tests, we are frequently looking to:
+Example using the command-line:
 
-* Locate tests
-* Write logs to a given location
-* Grab files that will be useful for tests, such as ISO files or VM disk
-  images
+.. code-block:: bash
 
-Avocado has a module dedicated to find those paths, to avoid cumbersome
-path manipulation magic that people had to do in previous test frameworks [#f1]_.
+  $ avocado run --foo bar /bin/true
 
```

Comment:
Then I guess using quotes here is also well-defined. It would definitely be needed if we want to specify space-containing string.